doc/source/timeseries.rst

```diff
@@ -557,6 +557,50 @@ We are stopping on the included end-point as it is part of the index
    dft2 = dft2.swaplevel(0, 1).sort_index()
    dft2.loc[idx[:, '2013-01-05'], :]
 
+.. versionadded:: 0.21.1
```

Comment:
add a sub-section label here (with a ref), call it something like ``slicing with timezones``.